ui/src/containers/SubOrgForm/index.js

```diff
@@ -477,6 +485,46 @@ class SubOrgForm extends Component {
               onDismiss={this.onExpirationDismiss}
               readonly={!this.props.isSiteAdmin} />
           </div>
+
+          <fieldset>
+            <legend className="pageHeader">Export Expiry</legend>
+            <div className="from-group">
+              <Checkbox
+                label="Don't Allow Exports"
+                style={styles.checkbox}
+                onChange={this.onChangeBooleanSetting.bind(
```

Comment:
Probably easier and more readable to wrap this in a function and just call `this.onChangeBooleanSetting`. As a rule I'd try avoid using `bind` wherever we can because new people don't totally understand what it does.

Reply:
Same is true further down too.

Reply:
https://github.com/LearningLocker/learninglocker/pull/1215/commits/cb1c60f3bbb19386d0aec469a1d346885a8a3141 removed bind